pages/resources/rules.tsx

```diff
@@ -15,29 +15,30 @@ export default function Home() {
-                                <h3 className="uppercase my-2 tracking-wide font-semibold text-gray-200 text-center md:text-left">Inappropriate Chat Conduct</h3>
+                                <h3 className="uppercase my-2 tracking-wide font-semibold text-gray-200 text-center md:text-left">Chat Based Rules</h3>
                                 <ol className="text-gray-300 w-full flex flex-col justify-center space-y-4 md:ml-6 list-inside list-decimal">
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Spam, flood, or excessive capital usage is not allowed.</li>
-                                    <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Posting a link not related to the ongoing chat. All Links posted should be <span className="rounded-md bg-gray-900 bg-opacity-50 p-1">https://</span> and not contain any viruses or otherwise malicious content.</li>
+                                    <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Posting a link not related to the ongoing chat is not allowed. All Links posted should begin with <span className="rounded-md bg-gray-900 bg-opacity-50 p-1">https://</span> and not contain any malicious content.</li>
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">NSFW content (nudity, gore, etc) or excessively inappropriate behavior is not allowed.</li>
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Disrespectful behavior is not allowed. Swearing is allowed, however directed or excessive swearing is not.</li>
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Perverse behavior and creating/continuing arguments for the sake of an argument are not allowed.</li>
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Arguing with or publicly flaming staff members based on their administrative actions is not allowed. Staff members will always have the final say - any complaints should be directed to <span className="rounded-md bg-gray-900 bg-opacity-50 p-1">Levah#0001</span> and not public chat.</li>
                                     <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Using our platforms to advertise other content (whether via DM, public chat, or ingame messaging channel [like a party chat]) is not allowed unless otherwise specified.</li>
-                                    <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Purposely misusing channels is not allowed. If you are unsure of a channel’s usage, ask.</li>
+                                    <li className="w-full md:w-[95%] px-3 py-4 bg-gray-800 rounded">Purposely misusing channels on the Discord is not allowed. If you are unsure of a channel’s usage, ask.</li>
```

Comment:
I actually said didn't add "in the discord" in purpose in case we added like "channels" ingame